jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -56,7 +56,7 @@ Optionally configure workflows, using the following instructions:
 
 1. To use the Workflows feature for job orchestration, first split your build job into multiple jobs, each with a unique name.
 
-2. To persist a job artifact, add the `persist_to_workspace:<directory>` key at the end of the job, under `steps:`. Then, to get the saved artifact for use by another job, add the `attach_workspace:` key with a nested `at:<directory>` key, ideally  defined as the directory where the artifact was saved.
+2. To persist a temporary file to a location between jobs, add the `persist_to_workspace:<directory>` key at the end of the job, under `steps:`. Then, to get the saved temporary file for use by another job, add the `attach_workspace:` key with a nested `at:<directory>` key, ideally defined as the directory where the artifact was saved.
```

Comment:
"at the end of the job" not necessary.  `persist_to_workspace` can be called whenever.

They can also be called multiple times (I think, don't quote me).